polaris-icons/icon-template.js

```diff
@@ -0,0 +1,25 @@
+const template = (
+  {template},
+  opts,
+  {imports, interfaces, componentName, props, jsx, exports},
+) => {
+  const plugins = ['jsx'];
+  if (opts.typescript) {
+    plugins.push('typescript');
+  }
+  const typeScriptTpl = template.smart({plugins});
+  return typeScriptTpl.ast`${imports}
```

Comment:
I don't have context on these kinds of things but this file seems confusing to me. With this unguarded return here is anything below even getting run?

Reply:
We use the default [template](https://github.com/gregberge/svgr/blob/v4.3.3/packages/babel-plugin-transform-svg-component/src/index.js#L3C1-L12C2) directly, so I don't want to change it more than we have to.